src/scenes/PlantingSitesRouter/view/BoundariesAndZones.tsx

```diff
@@ -78,53 +76,51 @@ export default function BoundariesAndZones({
         <Typography fontSize='16px' fontWeight={600} margin={theme.spacing(3, 0)}>
           {strings.BOUNDARIES_AND_ZONES}
         </Typography>
-        {adHocObservationSupportEnabled && view === 'list' && (
-          <Box display={'flex'} alignItems='center'>
```

Comment:
I think this one still needs to check that `view === 'list'` right?

Reply:
Oops, good catch! yes!